k8s/mock-awala-middleware.yml

```
@@ -14,3 +14,30 @@ spec:
           env:
             - name: MOCKSERVER_SERVER_PORT
               value: "8080"
+---
+apiVersion: sources.knative.dev/v1
+kind: SinkBinding
+metadata:
+  name: veraid-authority-awala-sink-binding
```

This is agnostic of VeraId Authority.

```suggestion
  name: mock-awala-middleware-sink-binding
```